src/js/components/DataTable/Body.js

```diff
@@ -165,6 +198,7 @@ const Body = forwardRef(
               }
             : undefined
         }
+        onTab={() => setActive(undefined)}
```

Comment:
I'm still unclear why we need `onTab` and can't do everything via `onFocus` and `onBlur`.

Reply:
Now moved to the DataTable-specific PR, but is now removed